### PR TITLE
DockerV2: Updating from docker-common to docker-common-v2

### PR DESCRIPTION
**Pull request by savenkat**
- Also moved from utility-common to utility-common-v2
- Bumped up patch version

### CODE REVIEW

File: Tasks/DockerV2/make.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (ds-ms) [inline Tasks/DockerV2/make.json]: Is this `make.json` missing `rm` step?

File: Tasks/DockerV2/package-lock.json [hunk 2]
```diff
@@ -67,22 +80,22 @@
...
-        "pify": "2.3.0",
-        "pinkie-promise": "2.0.1",
-        "rimraf": "2.6.3"
+        "globby": "^4.0.0",
+        "is-path-cwd": "^1.0.0",
+        "is-path-in-cwd": "^1.0.0",
+        "object-assign": "^4.0.1",
+        "pify": "^2.0.0",
+        "pinkie-promise": "^2.0.0",
+        "rimraf": "^2.2.8"
```
Comment (ds-ms) [inline Tasks/DockerV2/package-lock.json]: You may want to revert these changes or generate a package-lock.json without `^`s (compliance)

Comment (Anumita) [reply Tasks/DockerV2/package-lock.json]: +1, can use the changes in https://github.com/microsoft/azure-pipelines-tasks/pull/10569 to save the exact version

File: Tasks/DockerV2/make.json (hunk 1 above)
Comment (savenkat) [reply Tasks/DockerV2/make.json]: added

File: Tasks/DockerV2/package-lock.json (hunk 2 above)
Comment (savenkat) [reply Tasks/DockerV2/package-lock.json]: --save-exact doesn't work on its own with npm > 6.*
